data/human/deep missions.txt

```diff
@@ -2825,21 +2825,33 @@ mission "Home for Skadenga 1"
 			label questions
 			choice
 				`	"What happened to the researchers?"`
+					to display
+						not "skadenga: asked about researchers"
 				`	"Is it safe to drive over the ice?"`
+					to display
+						not "skadenga: asked about ice"
 					goto ice
 				`	"Are there sea-monsters beneath the ice?"`
+					to display
+						not "skadenga: asked about monsters"
 					goto monster
```

Comment:
Putting the `goto` before `to display` is more readable since all the text related to one option is lined up. If the `goto` is after `to display` then it looks like it is associated with the next choice.

```suggestion
					to display
						not "skadenga: asked about researchers"
				`	"Is it safe to drive over the ice?"`
					goto ice
					to display
						not "skadenga: asked about ice"
				`	"Are there sea-monsters beneath the ice?"`
					goto monster
					to display
						not "skadenga: asked about monsters"
```

I know from a programming perspective, you check if you display it before goto-ing to it, but from a readability prespective, the opposite is easier to follow.

Reply:
Look at the massive conversation in "Hai Reveal [A09] History of Appeasement" in this PR to see a larger version. It may be more apparent there. It's around line 1040 of this:

https://github.com/endless-sky/endless-sky/pull/7658/files#diff-73fb5ebd354e9467bdc3d6b9d357960d835f9e9740aabdab0d3197a389076d7e

Reply:
@Amazinite has suggested empty lines between the choices in complex blocks to help separate them.

We both prefer to list the conditions first because, when reading the data, whether or not an option should be displayed is probably going to be relevant before the `goto`, since the `goto` doesn't matter if it's not being displayed.

Reply:
That is not what is in the wiki. It has the "goto" before the "to display"

```
    choice
        <text>
            [<endpoint> | goto <label>]
            ["to display"]
                <condition>
```

Reply:
If it is written that way, then your eyes will quickly pick out both the goto and "to display," so there's really no issue of which one is first.

Reply:
I definitely had a hard time picking out which lines the `to display` were associated with at first. For me personally, I think I'd like that _first_, followed by the dialog line, then the goto. But that's probably not doable with the code as-is. But either way, when it's getting to 3+ lines of text, breaking it up with newlines does feel like the way to go, yeah.